packages/tezos-rust-libs/tezos-rust-libs.1.1/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "2.0"
+synopsis: "Tezos: all rust dependencies and their dependencies"
+maintainer: "contact@tezos.com"
+authors: "Tezos devteam"
+license: "multiple"
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos-rust-libs/issues"
+depends: ["conf-rust"]
+build: ["cargo" "build" "--target-dir" "target" "--release"]
```

Comment:
The previous version has this instead:
```
  ["mkdir" ".cargo"]
  ["mv" "cargo-config" ".cargo/config"]
  ["cargo" "build" "--target-dir" "target" "--release"]
```
Is it not needed anymore?

Reply:
Argh, my mistake and well caught, Yes this is still needed...

Reply:
Nice catch. I fixed in a second commit, with the same message than https://github.com/ocaml/opam-repository/pull/17880

Reply:
Mmh, wait. Not sure it is required. The file is at the correct place: see https://gitlab.com/tezos/tezos-rust-libs/-/commit/f0a269087fe86dbe1cf471bb76e42fb4a481e226.